dbscan/dbscan.go

```diff
@@ -210,25 +209,25 @@ func (api *API) processRows(dst interface{}, rows Rows, multipleRows bool) error
 			err = rs.Scan(dst)
 		}
 		if err != nil {
-			return errors.WithStack(err)
+			return fmt.Errorf("scanning: %w", err)
 		}
 		rowsAffected++
 	}
 
 	if err := rows.Err(); err != nil {
-		return errors.Wrap(err, "scany: rows final error")
+		return fmt.Errorf("scany: rows final error: %v", err)
```

Comment:
%v should be used here instead of %w IMO, more info in `Errors and package APIs` in https://go.dev/blog/go1.13-errors

Reply:
I disagree here. It makes more sense to allow to inspect the original error from the underlying library.

Reply:
ok, will wrap these